attest/verifier/types/src/verification.rs

```diff
@@ -23,10 +23,12 @@ use sha2::{Digest, Sha256};
 pub struct DcapEvidence {
     pub quote: Option<Quote3<Vec<u8>>>,
     pub collateral: Option<Collateral>,
+    pub report_data: Option<EnclaveReportDataContents>,
 }
 
 const TAG_DCAP_EVIDENCE_QUOTE3: u32 = 1;
 const TAG_DCAP_EVIDENCE_COLLATERAL: u32 = 2;
+const TAG_DCAP_EVIDENCE_REPORT_DATA: u32 = 3;
 
 // Quote3 and Collateral cannot trivially be made to implement prost::Message.
```

Comment:
I know this existed before this PR but I think mentioning why that is the case would be helpful for people looking at this in the future.

Reply:
Might also be worth mentioning why this needs Prost at all

Reply:
> I know this existed before this PR but I think mentioning why that is the case would be helpful for people looking at this in the future.

I'm not sure I follow:
Are you saying add some comments better describing the protobuf tag numbering, or are you saying we want to better describe the usage of protobof to serialize the `DcapEvidence` between endpoints?

Reply:
I'm requesting to mention why Quote3 and Collateral cannot trivially be made to implement prost::Message.

Reply:
Thanks!